src/html/HTMLWriter.jl

```diff
@@ -1790,6 +1795,25 @@ function write_html(ctx::HTMLContext, navnode::Documenter.NavNode, page_html::DO
     return true
 end
 
+"""
+Calculates and converts bytes to appropriate format.
+"""
+function format_units(size, unit = "bytes")
```

Comment:
It doesn't really make sense to pass this as an argument here.

```suggestion
function format_units(size)
    unit = "bytes"
```